### ISSUE TITLE
Improvement of the B2B sales process

### ISSUE THREAD

**Issue by sadortun** [orocrm/crm#221]
Hi Oro Team,

Here is a couple of suggestions to improve the sales process. Currently when creating leads and opportunities we need to repeat the same informations on many forms. This make it hard to use. This is the continuation of #179
# Creating a lead form

When the lead form is filled ( first name, last name ..... compagny name ... address .... 
When clicking on the `+` to create a **Contact** or a **Business account.** All the fields that are already fillled in the Lead Form should be used to pre-fill the **Contact**  or **Business account** form.

Also, there should have a simplified way of entering all the informations.

So, lets go step by step to crete a lead with the 3 informations :
- ACME Intl.
- John Doe
- john.doe@acme.intl

**This require the user to  fill 13 fields,**  in 4 different forms. Which is highly error prone. and hard to follow. I think that having a mecanisim to propagate the fields to the nested forms would be a great improvement. Unfortunatly, i am not familiar enough with the core to be able to submit a PR on this.

**Lead Form**
![image](https://cloud.githubusercontent.com/assets/794449/18937071/d93b47a8-85ba-11e6-9b38-e43ee76aa46f.png)

**Create Contact Form**
![image](https://cloud.githubusercontent.com/assets/794449/18937090/0648acf4-85bb-11e6-9934-d7de13662f15.png)

**Create Business Customer form**
![image](https://cloud.githubusercontent.com/assets/794449/18937117/36c19e90-85bb-11e6-9568-738aa8bb416c.png)

**Create Account Form**
![image](https://cloud.githubusercontent.com/assets/794449/18937137/6270c35e-85bb-11e6-9998-20c87da02915.png)

Thanks,
Samuel

**Comment by artml** [comment]
Hi Samuel,

That's definitely a great suggestion and I think we definitely will work on it in the future. Not in the upcoming release however—we see Opportunity management as more of a priority now—but definitely one in the future.

**Comment by sadortun** [comment]
I think this is mostly solved with the great improvements in 1.10.x and 2.x ! Thanks
